.github/workflows/data-prepper-peer-forwarder-local-node-e2e-tests.yml

```diff
@@ -26,4 +26,4 @@ jobs:
       - name: Checkout Data-Prepper
         uses: actions/checkout@v2
       - name: Run raw-span latest release compatibility end-to-end tests with Gradle
-        run: ./gradlew -PendToEndJavaVersion=${{ matrix.java }} :e2e-test:trace:rawSpanLatestReleaseCompatibilityEndToEndTest
+        run: ./gradlew -PendToEndJavaVersion=${{ matrix.java }} :e2e-test:peerforwarder:localAggregateEndToEndTest
```

Comment:
Perhaps we should keep the raw span compatibility test. We will want this when we start preparing for Data Prepper 2.1 because it should remain compatible.